shin-sforzando/etude-firebase
# feat: :+1: implement Firebase Auth

## Describe the PR

Authentication is implemented using Firebase.

## Screenshots

None.

# Detail of the change

A clear and concise description of what you changed.

- [x] Introduce Firebase JS SDK

## Anticipated impacts

None.

## To reproduce

Steps to check the behavior:

1. Update Libraries `yarn install`
1. Run Development Server `yarn dev`
1. Check on `http://0.0.0.0:3000/`

## Appendix

None.
